maps/southern_cross/structures/closets/misc_vr.dm

```diff
@@ -7,7 +7,7 @@
 	icon_opened = "secureexpopen"
 	icon_broken = "secureexpbroken"
 	icon_off = "secureexpoff"
-	req_access = list(access_explorer)
```

Comment:
Does pathfinder not have an access level...?

Reply:
Not their own, no. I never added one.